microsoft-edge/progressive-web-apps-chromium/how-to/digital-goods-api.md

```diff
@@ -35,17 +35,47 @@ The Payment Request API⁠⁠ handles the actual payment transaction when a purc
 See [Payment Request API](https://developer.mozilla.org/docs/Web/API/Payment_Request_API) at MDN.
 
 
+<!-- ====================================================================== -->
+## Enable the Digital Goods API
+
+The Digital Goods API is currently available in pre-release versions of Microsoft Edge.  To use the API, either enable the feature locally in Microsoft Edge, or register in the Digital Goods API origin trial to test the API in production.
+
+
+<!-- ------------------------------ -->
+#### Enable the feature locally
+
+To enable the Digital Goods API on your developer machine only, for testing:
+
+TODO: How do you enable the flags `msWebAppDigitalGoods`, `AppStoreBilling` and `DigitalGoodsApi`?  They are excluded from `edge://flags`.
+
+<!-- The Edge Experiments flag in `edge://flags/` will be enabled by default, when this feature ships in Edge Stable. -->
+<!-- todo: which flag? -->
+
+
+<!-- ------------------------------ -->
+#### Use the origin trial to test the feature in production
+
+To test the Digital Goods API in production, register in the origin trial.  See: 
+* [Digital Goods API](https://developer.microsoft.com/microsoft-edge/origin-trials/trials/4b4a9ead-d912-4349-87b3-25e5e50b4f13) at _Microsoft Edge Developer_ > _Origin Trials_.
+* [Registering for an origin trial](../../origin-trials/index.md#registering-for-an-origin-trial) in _Use origin trials in Microsoft Edge_.
+
+<!-- This feature ships with a pre-release of Microsoft Edge.  todo: specify the exact version number here when available -->
```

Comment:
This feature ships with a pre-release of Microsoft Edge. todo: specify the exact version number here when available

fresh PR Comment b/c previous Comment was Outdated
@runyuany

Reply:
```
To use this API, you must install a prerelease of Microsoft Edge, or a later release:
* Edge Dev 115.0.1860.0
* Edge Canary 115.0.1860.0
```